src/modules/addRemove.js

```diff
@@ -0,0 +1,212 @@
+import arrow from '../arrow-key.svg';
+import dots from '../dots.svg';
+import trash from '../trash.svg';
+
+// selecting the container
+const container = document.querySelector('.container');
+
+// selecting the input container
+const inputSection = document.querySelector('.inputSection');
+
+// selecting the input field
+const inputField = document.getElementById('inputField');
+
+// local storage
+const inputStorage = JSON.parse(localStorage.getItem('inputdata')) || {};
+inputField.value = inputStorage.inputField || '';
+
+const inputSave = () => {
+  inputStorage.inputField = inputField.value;
+
+  localStorage.setItem('inputdata', JSON.stringify(inputStorage));
+};
+
+const clearInput = () => {
+  inputStorage.inputField = '';
+
+  localStorage.setItem('inputdata', JSON.stringify(inputStorage));
+};
+
+// attach an event listener to the input field to listen for changes
+inputField.addEventListener('input', inputSave);
+
+// selecting the arrow for entry
+const arrowHolder = document.createElement('img');
+arrowHolder.classList.add('arrow');
+arrowHolder.src = arrow;
+
+inputSection.append(inputField, arrowHolder);
+
+// implementing list functionality
+
+// local storage for array
+let taskHolder = JSON.parse(localStorage.getItem('taskList')) || [];
+
+const bookSave = () => {
+  localStorage.setItem('taskList', JSON.stringify(taskHolder));
+};
```

Comment:
Great job @gisachris 🤝

- [ ] [Optional] Kindly ensure the property index is set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal 5).